app/javascript/lib/metrics-api.js

```diff
@@ -339,14 +356,19 @@ function getAnalyticsPageName() {
  * @param {String} name
  * @param {Object} props
  */
-export function log(name, props={}) {
+export function log(name, props = {}) {
   props = Object.assign(props, {
     appId: 'single-cell-portal',
     appPath: getAnalyticsPageName(),
     appFullPath: getAppFullPath(),
     env
   }, getDefaultProperties())
 
+  const tab = getTabProperty()
+  if (tab) {
+    props['referringTab'] = tab
```

Comment:
Blocker: We use `referringDomain` to mean "the _previous_ domain".  I could see analysts interpreting "referring tab" as "the previous tab"; that's how I first interpreted it.

So for consistency and simplicity, I think we should go with:

```suggestion
    props['tab'] = tab
```

Reply:
I went back and forth on this, because if you click from one tab to another in the wizard the tab you clicked _from_ gets logged as `referringTab` so I thought it would make it less confusing for that, but I see what you mean about that being less clear for button clicks within a tab. I'll look into making this most clear